pitch.md

```diff
@@ -23,38 +35,187 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+been played in a non optimal manner.
+
+I believe this is data description computational model since it is mainly displaying
+the information the user gave it or pieces of information the user gave it in
+nice ways.
+
+A common syntax error that you will find in bridge books is players not having
+the correct number of cards.  It is not uncommon to accidently see a misprint
+where a player will have a 14 card hand (should be 13). I think an error message
+should be given that not all players have the same number of cards.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+Inputting and then viewing hands should be very easy with this langauge.  Being
+able to do some basic analysis should also fairly easy.  Additionally, showing an
+piece of a hand or showing the hand in multiple states (such as at the begining
+and after the first 8 tricks have been played) should be easy.
+
+The intention is to use this for bridge hands so if one wanted to use this language
+to describe some other game that is similar, such as say hearts or spades, it
+might be difficult or impossible.
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+[BridgeBase](http://www.bridgebase.com) offers the ability to export hands in a
+.lin format, however this is hard to read.  Here is an example of a hand I
+played: 
+> pn|guplex,~~M12054,~
+> ~M12052,~~M12053|st||md|1S58KAH269KDTKC34Q,S26JQH4D246JAC267,S349H37QAD3789C5A
+> ,|rh||ah|Board 11|sv|o|mb|1N|an|notrump opener. Could have 5M. -- 2-5
+> !C|mb|p|mb|2C|an|Stayman --   |mb|p|mb|2H|an|2-5 !C; 2-5 !D; 4-5 !H; 2-4 !S;
+> 15-17 HC|mb|p|mb|4H|an|4+ !H; 10-13 total points |mb|p|mb|p|mb|p|pg||pc|C2|pc|
+> C5|pc|CK|pc|C3|pg||pc|S7|pc|SA|pc|S6|pc|S3|pg||pc|C4|pc|C6|pc|CA|pc|CJ|pg||pc|
+> D3|pc|DQ|pc|DK|pc|DA|pg||pc|SQ|pc|S4|pc|ST|pc|SK|pg||pc|H2|pc|H4|pc|HA|pc|H5|p
+> g||pc|H3|pc|HT|pc|HK|pc|D4|pg||pc|CQ|pc|C7|pc|S9|pc|C8|pg||pc|DT|pc|DJ|pc|D7|p
+> c|D5|pg||pc|S2|pc|HQ|pc|C9|pc|S5|pg||pc|D8|pc|H8|pc|H9|pc|D6|pg||pc|S8|pc|SJ|p
+> c|H7|pc|CT|pg||pc|D9|pc|HJ|pc|H6|pc|D2|pg||
+
+I think this could be more readable to humans.  Also I have difficulty finding
+a way to process this data without writing something myself. It is nice that
+you can do double dummy analysis and click through the tricks in the order that
+they were played if you upload a .lin file to bridgebase. The main thing that this
+doesnt offer is an easy way to share this hand with friends unless they want to
+be sent the file and then upload it to bridgebase.  I would like to have a way
+to output in an easy to read manner the hand and send it via email or social
```

Comment:
I agree that being able to share the information will be really interesting and important. I also think it will be interesting to understand how you want this to be formatted. If it's plaintext, you may need to check how different mediums format their text (does the same formatting look different on a Facebook post versus an email?). It may be more consistent but clunkier to have an image of the plaintext, so this will be an interesting design decision.